jaxlib/lapack.pyx

```diff
@@ -59,13 +59,14 @@ cdef void blas_strsm(void* out, void** data) nogil:
   cdef int32_t diag = (<int32_t*>(data[3]))[0]
   cdef int m = (<int32_t*>(data[4]))[0]
   cdef int n = (<int32_t*>(data[5]))[0]
-  cdef float* alpha = <float*>(data[6])
-  cdef float* a = <float*>(data[7])
-  cdef float* b = <float*>(data[8])
+  cdef int num_b = (<int32_t*>(data[6]))[0]
```

Comment:
I might call this `batch` rather than `num_b`, since there's another `b` in this scope. Same everywhere.

Reply:
have done!